apps/portal/components/pagesComponents/_videoEditorScreen/SingleVideoPageManageAreaTemplate-old.tsx

```
@@ -499,21 +504,26 @@ const SingleVideoPageManageAreaTemplate: React.FC<IProps> = ({
             </Tab>
           </TabList>
           <TabPanel>
             <ImageActionsCard className="tw-rounded-t-none">
               <div>
-                Currently saved in
-                <strong className="tw-ml-1.5">{currentlySavedIn}</strong>
+                <Trans
+                  values={{ currentlySavedIn: currentlySavedIn }}
+                  i18nKey="page.image.currentlySavedInOnly"
+                  components={{
+                    strong: <strong></strong>,
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)